src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -2071,6 +2071,12 @@ def _ensure_default_log_analytics_workspace_for_monitoring(cmd, subscription_id,
     workspace_region = "eastus"
     workspace_region_code = "EUS"
 
+    # sanity check that locations and clouds match.
+    if ((cloud_name.lower() == 'azurecloud' and AzureChinaRegionToOmsRegionMap.get(rg_location, False)) or
```

Comment:
nit, `rg_location in AzureChinaRegionToOmsRegionMap` might save a few characters here.